index.ts

```diff
@@ -82,13 +82,77 @@ function makeHash(viewFunction: string): string {
     return createHash("MD5").update(viewFunction).digest("hex");
 }
 
+export interface ViewFunctionConfig {
+    viewFunction: string;
+    model?: any;
+    decoder?: string;
+    indent?: number;
+    newLines?: boolean;
+}
+
+// compiles multiple view functions into one elm file
+// which is much faster if you're likely to need all of them
+export function grouped(

```

Comment:
🎨  the name `grouped` doesn't make much sense to me, though I know why it's called that. How about something like `elmStaticHtml.multiple()` instead?